bootstrap/build.cmd

```diff
@@ -0,0 +1,21 @@
+@ECHO off
+SETLOCAL
+
+SET KOREBUILD_REPO=https://github.com/aspnet/KoreBuild
+SET KOREBUILD_BRANCH=victorhu/init
+
+SET SCRIPT_FOLDER=%~dp0
+SET KOREBUILD_SCRIPT=%SCRIPT_FOLDER%\KoreBuild.cmd
+
+if EXIST "%KOREBUILD_SCRIPT%" GOTO endbootstrap
+
+SET TEMP_CLONE_LOCATION=%TEMP%\KoreBuildClone
```

Comment:
Multiple builds on same machine shouldn't conflict. Use a unique name (which is easier to do in PowerShell).